extensions/ql-vscode/src/view/model-editor/HiddenMethodsRow.tsx

```diff
@@ -0,0 +1,30 @@
+import {
+  VSCodeDataGridCell,
+  VSCodeDataGridRow,
+} from "@vscode/webview-ui-toolkit/react";
+import * as React from "react";
+import { styled } from "styled-components";
+
+const HiddenMethodsCell = styled(VSCodeDataGridCell)`
+  text-align: center;
+`;
+
+interface Props {
+  numHiddenMethods: number;
+  someMethodsAreVisible: boolean;
+}
+
+export function HiddenMethodsRow(props: Props) {
+  if (props.numHiddenMethods === 0) {
+    return null;
+  }
+
+  return (
+    <VSCodeDataGridRow>
+      <HiddenMethodsCell gridColumn="span 5">
+        {props.someMethodsAreVisible && "And "}
+        {props.numHiddenMethods} methods modeled in other CodeQL packs
```

Comment:
Could we use [`pluralize`](https://github.com/github/vscode-codeql/blob/9925b4e3bcf7544a222a5fa7b80b7297da535060/extensions/ql-vscode/src/common/word.ts#L6) here to ensure we use "method" in case there's only 1 method?